mysqldef/fsp.go

```diff
@@ -80,3 +80,13 @@ func parseFrac(s string, fsp int) (int, error) {
 	//  0.0312 round 2 -> 3 -> 30000
 	return int(round * math.Pow10(MaxFsp-fsp)), nil
 }
+
+// alignFrac is used to generate alignment frac, like `100` -> `100000`
+func alignFrac(s string, fsp int) string {
```

Comment:
it will be treated as number, like `10000000`,  means 10000000 microseconds, but `100` means 100000 microseconds.